omod/src/main/webapp/fragments/header.gsp

```diff
@@ -114,15 +114,16 @@
 
 </script>
 <header>
-    <div class="logo">
-        <a href="${ logoLinkUrl }">
-            <img src="${ logoIconUrl }"/>
-        </a>
-    </div>
+
     <% if (context.authenticated) { %>
 
         <% if (useBootstrap) { %>
             <nav class="navbar navbar-expand-lg navbar-dark navigation">
+                <div class="logo">
+		    <a href="${ logoLinkUrl }">
+		    <img src="${ logoIconUrl }"/>
+		    </a>
+		</div>
```

Comment:
Could you change the indentation here to match. In particular, this `gsp` seems to be using spaces instead of tabs. You may need to configure your editor accordingly.

Reply:
Will do that